Assets/AnkrSDK/Examples/Scripts/ERC20Example/ERC20Example.cs

```diff
@@ -94,6 +91,7 @@ public async void GetEvents()
 			{
 				fromBlock = BlockParameter.CreateEarliest(),
 				toBlock = BlockParameter.CreateLatest(),
+				filterTopic2 = new [] { EthHandler.DefaultAccount }
```

Comment:
why 2 and not 1?

Reply:
Because second topic it's a user address in that event.